modules/wri_paragraph/wri_paragraph.install

```diff
@@ -0,0 +1,171 @@
+    'translatable',
+  ], 'wri_paragraph');
+  \Drupal::service('distro_helper.updates')->updateConfig('field.field.paragraph.impact_in_action_blocks.field_block', [
+    'translatable',
+  ], 'wri_paragraph');
+  \Drupal::service('distro_helper.updates')->updateConfig('field.field.paragraph.listing_block.field_block', [
+    'translatable',
+  ], 'wri_paragraph');
+  \Drupal::service('distro_helper.updates')->updateConfig('field.field.paragraph.narrative_taxonomy.field_block', [
+    'translatable',
+  ], 'wri_paragraph');
+  \Drupal::service('distro_helper.updates')->updateConfig('field.field.paragraph.partner_logo.field_link', [
+    'translatable',
+  ], 'wri_paragraph');
+  \Drupal::service('distro_helper.updates')->updateConfig('field.field.paragraph.partner_logo.field_partner_logo', [
+    'translatable',
+  ], 'wri_paragraph');
+  \Drupal::service('distro_helper.updates')->updateConfig('field.field.paragraph.person_listing.field_block', [
+    'translatable',
+  ], 'wri_paragraph');
+  \Drupal::service('distro_helper.updates')->updateConfig('field.field.paragraph.program_lead_in.field_leadin_paragraph', [
+    'translatable',
+  ], 'wri_paragraph');
+  \Drupal::service('distro_helper.updates')->updateConfig('field.field.paragraph.program_lead_in.field_leadin_title', [
+    'translatable',
+  ], 'wri_paragraph');
+  \Drupal::service('distro_helper.updates')->updateConfig('field.field.paragraph.side_by_side.field_body', [
+    'translatable',
+  ], 'wri_paragraph');
+  \Drupal::service('distro_helper.updates')->updateConfig('field.field.paragraph.side_by_side.field_title', [
+    'translatable',
+  ], 'wri_paragraph');
+  \Drupal::service('distro_helper.updates')->updateConfig('field.field.paragraph.small_cta.field_block', [
+    'translatable',
+  ], 'wri_paragraph');
+  \Drupal::service('distro_helper.updates')->updateConfig('field.field.paragraph.statistic.field_statistic_number', [
+    'translatable',
+  ], 'wri_paragraph');
+  \Drupal::service('distro_helper.updates')->updateConfig('field.field.paragraph.statistic.field_statistic_text', [
+    'translatable',
+  ], 'wri_paragraph');
+  \Drupal::service('distro_helper.updates')->updateConfig('field.field.paragraph.statistics.field_statistics', [
+    'translatable',
+  ], 'wri_paragraph');
+  \Drupal::service('distro_helper.updates')->updateConfig('field.field.paragraph.statistics.field_statistics_title', [
+    'translatable',
+  ], 'wri_paragraph');
+  \Drupal::service('distro_helper.updates')->updateConfig('field.field.paragraph.superfeatured_item.field_block', [
+    'translatable',
+  ], 'wri_paragraph');
+  \Drupal::service('distro_helper.updates')->updateConfig('field.field.paragraph.unique_feature_cta.field_block', [
+    'translatable',
+  ], 'wri_paragraph');
+  \Drupal::service('distro_helper.updates')->updateConfig('language.content_settings.paragraph.featured_statement', [
```

Comment:
I think that since these are new config files, they should just be `installConfig()`? This was part of the issue of new paragraphs not getting added to translated pages I believe. (Don't make the node's entity_refrence_revisions field translatable, make the paragraph itself translatable) 

I switched all of them to installConfig(), and everything else either got moved to another module (where I added an update hook) or removed for non-applicability.

Reply:
Perfect.